tabular/src/autogluon/tabular/models/fastainn/hyperparameters/searchspaces.py

```diff
@@ -34,3 +36,9 @@ def get_searchspace_multiclass(num_classes):
 
 def get_searchspace_regression():
     return get_searchspace_binary()
+
+
+def get_searchspace_quantile():
+    spaces = get_searchspace_regression()
+    spaces.update({'alpha': Categorical(0.001, 0.01, 0.1, 1.0)})
```

Comment:
A comment would be helpful here to (briefly) describe alpha

Reply:
Yes added some descriptions